faqs/galaxy/datasets_download_datasets.md

```diff
@@ -0,0 +1,11 @@
+---
+title: Download datasets
+area: datasets
+box_type: tip
+layout: faq
+contributors: [jennaj, Melkeb]
+---
+
+
+1. Opening the datasets found under history
+2. Clicking on the save icon {% icon galaxy-save %} inside the dataset
```

Comment:
Could you use consistent language in all the FAQs? Using imperative sentences (so "Click" instead of "Clicking")